examples/blame.rs

```diff
@@ -0,0 +1,88 @@
+extern crate git2;
```

Comment:
Was this example drawn from libgit2's `examples/blame.c`? If so, could you be sure to include the license block at the top?